integration-testing/integration-tests/run-jobs-integration-test.sh

```diff
@@ -60,14 +60,20 @@ then
     PROC2_PID=$!
     $NODE $INT_TEST_DIR/jobs-integration-test.js -H $HOSTNAME  -f $CERT_DIR -t1 --debug=true -T $TEST_TAG | tee $PROC1_OUTFILE &
     PROC1_PID=$!
-else
+elif [ $AUTHENTICATION_TYPE"" == "websocket" ]
+then
     echo "###################################################################"
     echo ${0##*/}": running jobs integration test (websocket/sigv4)"
     echo "###################################################################"
     $NODE $INT_TEST_DIR/jobs-integration-test.js -H $HOSTNAME -P=wss -t2 --debug=true -T $TEST_TAG | tee $PROC2_OUTFILE &
     PROC2_PID=$!
     $NODE $INT_TEST_DIR/jobs-integration-test.js -H $HOSTNAME -P=wss -t1 --debug=true -T $TEST_TAG | tee $PROC1_OUTFILE &
     PROC1_PID=$!
+else
```

Comment:
The jobs integration tests don't seem to support custom auth and it feels pretty redundant anyways.